index.html

```diff
@@ -65,6 +65,79 @@ <h2>Introduction</h2>
       TODO(henbos): Fill in.
     </p>
   </section>
+  <section id="*">
+    <h3>
+      <dfn>RTCRtpReceiver</dfn> extensions
+    </h3>
+    <p>
+      Let <code>RTCRtpReceiver</code> objects have an
+      <dfn>[[\ReceiverJitterBufferDelayHint]]</dfn> internal slot initially
+      initialized to <code>null</code>.
+    </p>
+    <pre class="idl">partial interface RTCRtpReceiver {
+  attribute double? jitterBufferDelayHint;
+};</pre>
+    <section>
+      <h2>Attributes</h2>
+      <dl data-link-for="RTCRtpReceiver" data-dfn-for="RTCRtpReceiver"
+      class="attributes">
+        <dt><code><dfn>jitterBufferDelayHint</dfn></code> of type <span class=
+        "idlAttrType"><a>double</a></span>, nullable</dt>
+        <dd>
+          <p>This attribute affects how much additional delay the system adds
+          before decoding and playout delays, by adjusting the length of the
+          jitter buffer using a <i>hint</i>. This allows the application to
+          influence the tradeoffs between having a longer or shorter jitter
+          buffer. The longer the buffer, the more chance there is that packets
+          arrive in time for playout, but the higher the delay is. The shorter
+          the buffer, the higher the risk of stuttering, but the shorter the
+          playout delay. For interactive applications, low delay may be
+          important, whereas for passive applications a stutter-free experience
+          may be more important.</p>
+          <p>Note that the delay is only a hint; a soft target. User agent will
+          maintain it by inserting or removing samples into the media but not
```

Comment:
Until reading further it sounds like these two sentences are contradicting each other:
"the delay is _only a hint; a soft target_. [The] user agent _will maintain it_..."

Should this say something closer to?

The user agent must attempt to achieve the target by inserting or removing samples into the jitter buffer, but it SHOULD NOT distort the media to the point of hurting user experience.

Reading these sentences makes me wonder: What does it mean to "insert or remove samples"? Does inserting only refer to samples synthesized based on existing data, or what happens when the buffer is empty? Can you only remove? In https://github.com/w3c/webrtc-stats/pull/408 I believe we discussed deceleration by means of combining samples is also a valid strategy, and we defined the acceleration and deceleration metrics in terms of the difference between samples received and samples played out - making it up to the implementation how an increase or decrease in samples is achieved.

Do you have to insert samples in order to achieve the target? Can you have a large buffer as per target but not use all of it? How do you decide if samples need to be inserted or not?

Reply:
As you said we have original waveform/samples from packets, this is our baseline. And through transformations we can make more or less samples out of it. If we have more samples, it is deceleration, if we have less samples then this is acceleration. That is what I meant by "insert or remove samples". How we do it is an implementation detail (by expanding vowels for e.g. as far as I remember). Also we have some sound (noise) generation there for special cases.

How about:
"The user agent must attempt to achieve the target by accelerating or decelerating media, but it SHOULD NOT distort the media to the point of hurting user experience." ?